loggingx/logger.go

```diff
@@ -33,6 +33,10 @@ func InitLogger(appName string, cfg Config) *zap.SugaredLogger {
 		lgrCfg.Level = zap.NewAtomicLevelAt(zap.InfoLevel)
 	}
 
+	if !cfg.Stacktrace {
+		lgrCfg.DisableStacktrace = true
+	}
```

Comment:
The current behavior is to include stack traces.
It looks like this would change once this has been implemented to disable stack traces by default since `cfg.Stacktrace` if unconfigured would be false.

If we wanted wanted such a feature, it might make sense to change the configuration to a negation. Such as `cfg.DisableStacktraces` that way we aren't changing existing behavior.

Reply:
the default is set to true?

Reply:
ah yes, I did overlook that. The default values in flags is something we don't use, as they override vipers ability to check for configuration from environment variables or config files.

The requested changes above still hold. Even with the default set to true. A flag of `--stacktrace` sounds like you're enabling stacktraces but in fact you're disabling them. so the flag should be `--disable-stacktrace` and the accompanying config should also be `DisableStacktrace`.

Reply:
ok, ill flip that.

Reply:
done